services/order.py

```diff
@@ -0,0 +1,40 @@
+from django.db import transaction
+from django.contrib.auth import get_user_model
+
+
+from db.models import Order, Ticket
+
+
+User = get_user_model()
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = User.objects.get(username=username)
+    new_data = {}
+    if date:
+        new_data["created_at"] = date

```

Comment:
no need to create a dict for just 1 field

Reply:
It's ain't work into another way 😭